newIDE/app/src/InstructionOrExpression/EnumeratedInstructionOrExpressionMetadata.js

```diff
@@ -66,10 +65,16 @@ export const filterEnumeratedInstructionOrExpressionMetadataByScope = <
         eventsFunctionsExtension.getName(),
         eventsBasedBehavior.getName()
       ) === behaviorMetadata.getName()
-    ) {
+    )
+      return true;
+
+    // Show private non-behavior functions when editing the extension
+    if (
+      !behaviorMetadata &&
+      eventsFunctionsExtension &&
+      eventsFunctionsExtension.getName() === type.split(separator)[0]
```

Comment:
Why is this line needed?

Reply:
To check if the eventsFunctionExtension is the one the instruction or expression is from?